lib/services/local/configuration.go

```diff
@@ -19,21 +19,25 @@ package local
 import (
 	"context"
 
+	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/lib/backend"
 	"github.com/gravitational/teleport/lib/services"
 
 	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
 )
 
 // ClusterConfigurationService is responsible for managing cluster configuration.
 type ClusterConfigurationService struct {
 	backend.Backend
+	log logrus.FieldLogger
 }
 
 // NewClusterConfigurationService returns a new ClusterConfigurationService.
 func NewClusterConfigurationService(backend backend.Backend) *ClusterConfigurationService {
 	return &ClusterConfigurationService{
 		Backend: backend,
+		log:     logrus.WithField(trace.Component, "ClustrCfg"),
```

Comment:
Clust**e**r

Reply:
This was intentional because the tags displayed in the logs get truncated to exactly 9 characters and this seemed to be the most informative way of using that space.

Reply:
I think it's 11 actually? https://github.com/gravitational/teleport/blob/master/vendor/github.com/gravitational/trace/log.go#L51

Which would be exactly enough for `ClusterConf` :) This is not such a big deal though so if it indeed gets truncated, I think we can live with it.

Reply:
Right, but doesn't that count also the two brackets? :)

Reply:
Oh lol you're probably right. Let's just keep it as is, not worth such a lengthy discussion :)